examples/client/Locomotion/src/pages/AuthScreens/AuthLoadingScreen.js

```diff
@@ -90,6 +91,7 @@ const AuthLoadingScreen = () => {
       navigationService.replace(MAIN_ROUTES.START);
     }
 
+    await networkInfo.fetchData();
```

Comment:
why do we need this if the class already does it internally?

Reply:
Because I want also to fetch data on app initialization, I can remove it from the constructor